exercises/cookies/problem.md

```diff
@@ -1,60 +1,53 @@
-Create a server that has a route configuration exposing an endpoint ``set-cookie`` and ``check-cookie`` which can be access using 'GET' request. Specifically:
+Create a server that has a route configuration exposing an endpoint ``set-cookie`` and ``check-cookie`` which can be access using `'GET'` request. Specifically:
```

Comment:
Should it be: 
'which can be accessed using a `'GET'` request. 

I know this isn't what the PR is addressing, but what do you think? (everything else looks good)